src/bundle/Registry/CronJobsRegistry.php

```diff
@@ -45,12 +50,13 @@ public function __construct(string $environment, SiteAccess $siteaccess)
         $this->siteaccess = $siteaccess;
     }
 
-    public function addCronJob(Command $command, string $schedule = null, string $category = self::DEFAULT_CATEGORY): void
+    public function addCronJob(Command $command, string $schedule = null, string $category = self::DEFAULT_CATEGORY, string $options = ''): void
     {
-        $command = sprintf('%s %s %s --siteaccess=%s --env=%s',
+        $command = sprintf('%s %s %s%s --siteaccess=%s --env=%s',
```

Comment:
Missing space?
```suggestion
        $command = sprintf('%s %s %s %s --siteaccess=%s --env=%s',
```

Reply:
This is already taken into account in CronJobCompilerPass by prefixing the options value with an space if it is not empty.

Reply:
I would keep it in a proper form here. Also this will make the fallback unnecessary and will simplify the code a bit.